### PR TITLE
Multiple controller icon fixes

### PR DESCRIPTION
**Pull request by HaSa1002**
- Fix controller icons running in editor
- Fix controller icons not fitting the font size

Just some two tiny issues I wanted to get fixed.

### CODE REVIEW

**Comment by DasCapschen** [comment]
That's quite a lot of seemingly unrelated changes 🤔 

Also you've got conflicts.

**Comment by HaSa1002** [comment]
Ok gimme like ten minutes. I'll give you some more commits and actually start the branch on the correct base

**Comment by HaSa1002** [comment]
I did not fix the resource noise. I am going to leave that for my other PR

File: src/addons/controller_icons/objects/Sprite.gd [hunk 1]
```diff
@@ -21,15 +20,8 @@ func _on_input_type_changed(input_type):
...
-			texture = ControllerIcons.parse_path(path)
+		texture = ControllerIcons.parse_path(path)
 
 func set_show_only(_show_only: int):
 	show_only = _show_only
 	_on_input_type_changed(ControllerIcons._last_input_type)
-
-func set_force_type(_force_type: int):
-	force_type = _force_type
-	_on_input_type_changed(ControllerIcons._last_input_type)
```
Comment (DasCapschen) [inline src/addons/controller_icons/objects/Sprite.gd:35]: What's up with all this? And all the removed `tool` scripts? Is this right?

**Review by DasCapschen — APPROVED** [review]
lgtm

File: src/addons/controller_icons/objects/Sprite.gd (hunk 1 above)
Comment (HaSa1002) [reply src/addons/controller_icons/objects/Sprite.gd:35]: Yeah the author basically added an editor preview to see your labels and icons change but that generates a lot of noise and relies on hacks to work. So better deactivate all of that. I have no why force_type is not in there but we don't use it so I removed that code.

Comment (DasCapschen) [reply src/addons/controller_icons/objects/Sprite.gd:35]: Alright, sound good.

**Pull request merged**